### ISSUE TITLE
[QUESTION] Is Geocode working properly? 

### ISSUE THREAD

**Issue by CarolinaNazareth** [twintproject/twint#1102]
Hello!

I've been trying to extract tweets using UK geocode and it's actually pulling Tweets from all around the world. I'm not sure if I'm doing something wrong (I'm super new on this, please bear this in mind xD) of if the geocode feature is not working properly.

I'm running the code below using Jupyter Notebooks on a Mac and I've installed Twint using Github clone.

`config = twint.Config()
geocode='51.5074,0.1278'
config.Search = "we are hiring" 
config.Limit = 200
config.Store_csv = True
config.Output = "test_out_13.csv"
Location = True

twint.run.Search(config)`

Thanks in advance for your help. 🙏🏻

**Comment by innocentius** [comment]
Nope, geocode and location as far as i know is not working currently.

**Comment by mohammedayub44** [comment]
@innocentius Any idea if this will be fixed, I'm planning to search terms in different metro cities in US. I was planning on using -g --geo flag to filter for different cities. I'm guessing this will not work given geocode is not wokring. I tried running some queries on terminal it just gives me `No more data! Scraping will stop now`
